Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM openshift/origin-release:golang-1.13 AS builder
+WORKDIR ${GOPATH}/src/knative.dev/eventing-operator
+COPY . .
+ENV GOFLAGS="-mod=vendor"
+RUN go build -o /tmp/manager ./cmd/manager
+RUN cp -Lr ${GOPATH}/src/knative.dev/eventing-operator/cmd/manager/kodata /tmp
+
+FROM openshift/origin-base
+COPY --from=builder /tmp/manager /ko-app/eventing-operator
+COPY --from=builder /tmp/kodata/ /var/run/ko
+ENV KO_DATA_PATH="/var/run/ko"
+LABEL \
+    com.redhat.component="openshift-serverless-1-tech-preview-knative-eventing-rhel8-operator-container" \
+    name="openshift-serverless-1-tech-preview/knative-eventing-rhel8-operator" \
+    version="v0.13.0" \
```

Comment:
is that the proper version ?

Reply:
@matzew Took it from https://github.com/openshift-knative/serving-operator/commit/fa90677d2016f6593eb6ab116c232fbcdb611ee6

Reply:
cc @markusthoemmes

Reply:
I mainly asked b/c eventing itself has now 0.13.3 :-) 

but this version seems to be ok...

/lgtm